Makefile

```diff
@@ -189,6 +197,23 @@ $(KUSTOMIZE):
 	rm -rf $$KUSTOMIZE_GEN_TMP_DIR ;\
 	}
 
+.PHONY: opm
+opm: $(OPM)
+
+$(OPM):
+	@{ \
+	set -e ;\
+	OPM_GEN_TMP_DIR=$$(mktemp -d) ;\
+	cd $$OPM_GEN_TMP_DIR ;\
+	export GOPATH=$${OPM_GEN_TMP_DIR} ;\
+	go get github.com/operator-framework/operator-registry || true;\
+	cd src/github.com/operator-framework/operator-registry ;\
+	git checkout -b v1.15.1 ;\
+	make bin/opm ;\
+	mv bin/opm $@ ;\
+	rm -rf $$OPM_GEN_TMP_DIR ;\
+	}
```

Comment:
You could also fetch the release instead of building OPM. This is what we did in the compliance-operator. https://github.com/openshift/compliance-operator/blob/master/Makefile#L164

If you want, I could propose a similar patch here.

Reply:
I originally wanted to do that, but I did not like how the opm release binaries do not at least have a hash to provide some basic level of confidence since this would blindly install `opm` when building the bundle index image. I guess I feel more confident building from source than trusting a release binary without a hash. I filed https://github.com/operator-framework/operator-registry/issues/526 to request it.

Reply:
Got it. That's alright. Let's merge this as-is then. It can be changed once they address your bug report.